### ISSUE TITLE
feature request RemoveChatButtons

### ISSUE THREAD

**Issue by shamshitty** [BleedingBD/plugin-RemoveChatButtons#6]
so this is just a feature request for the voice category,
when you hover over user's avatars in a vc window, buttons at the bottom appear, it would be a nice feature if the options in the voice category *also* hid those buttons and not only in the panel at the bottom of the channel list as i think they should be included in my opinion ¯\_(ツ)_/¯
![image](https://user-images.githubusercontent.com/68004605/213813364-23b3226a-a76c-424d-901c-72e313c2cf30.png)

**Comment by QbDesu** [comment]
Yeah looking into this the only buttons that can easily be removed via CSS are the Screenshare and the Disconnect button.  Which wouldn't be enough for me to warrant putting support for just these in.
There are still other ways of removing them with JS instead of CSS, but for now I will not go that route with this plugin.
I'm not saying this won't ever be implemented, but not now.

**Comment by Banaanae** [comment]
Not sure if this should be a new issue or fits in here but other buttons like super reaction are really annoying
![image](https://user-images.githubusercontent.com/83927639/225259663-f5c126b0-58cc-4c4a-ae15-ada334e2dfc2.png)
